File: src/main/java/com/unihogsoft/scalag/vulkan/command/OneTimeCommandPool.java
package com.unihogsoft.scalag.vulkan.command;


import com.unihogsoft.scalag.vulkan.core.Device;

import static org.lwjgl.vulkan.VK10.VK_COMMAND_POOL_CREATE_TRANSIENT_BIT;

/**
 * @author MarconZet
 * Created 13.04.2020
 * Copied from Wrap
 */
public class OneTimeCommandPool extends CommandPool {
    public OneTimeCommandPool(Device device, Queue queue) {
        super(device, queue);
    }

    @Override
    protected int getFlags() {
        return VK_COMMAND_POOL_CREATE_TRANSIENT_BIT;
    }
}

File: src/main/scala/com/scalag/vulkan/command/OneTimeCommandPool.scala
package com.scalag.vulkan.command;

import com.scalag.vulkan.core.Device;

import org.lwjgl.vulkan.VK10.VK_COMMAND_POOL_CREATE_TRANSIENT_BIT;

/** @author
  *   MarconZet Created 13.04.2020 Copied from Wrap
  */
class OneTimeCommandPool(device: Device, queue: Queue) extends CommandPool(device, queue) {
  protected def getFlags: Int = VK_COMMAND_POOL_CREATE_TRANSIENT_BIT

}
